local_pytest.wst

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
What is this file for?

Reply:
I was planning to have all of the local tests set up in one place, but only have aims so far. Similarly to `complete_pytest.tin`. Maybe somewhere in `tests` would be more appropriate?

Reply:
Can you give it an even more explicit name? I agree that `tin` isn't the best (that's the name for our cluster), but `workstation` is even more generic. Either use the machine you plan for it to be run on, or, if it's really specific to your individual setup (i.e. you had to install things yourself and they're in your bin directory), your name?

Reply:
Adding my name would probably be the most appropriate here (done).

Reply:
Why is the directory named "complete"? If you just mean the calculators where the simplification is complete, maybe put that in the _script_ name? Are you envisioning other things that would belong in the "complete" subdir?  Also, if that file is meant to be kept long term, maybe we should make a `tests/local_scripts` subdirectory, and move the `.tin` and your script into there.

Reply:
Looks good.  Should I merge if I'm still happy with everything?

Reply:
Please do!